### PR TITLE
Set Up linter

### PR DESCRIPTION
**Pull request by RolandM99**
1. Set up Github actions ( add .github folder)
2. Add linter for HTML & CSS
3. Add .gitignore file
4. Add Html and Css file

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @RolandM99, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Hello Microverse</title>
+</head>
+<body>
+    <header>
+        <h1>Hello Microverse!</h1>
```
Comment (mke2111) [inline index.html]: - [ ] Add a class to this element so that you can use it to highlight and apply styles in the CSS file.

Comment (RolandM99) [reply index.html]: Okay, let me do it right!

File: README.md [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Major languages: HTML/CSS
+- Technologies used: Linter/Git/Github
 
 ## Live Demo
```
Comment (TedLivist) [inline README.md:16]: - [ ] **[OPTIONAL]** The demo link is broken. You can set up the live link like this:
- Go to this link (https://github.com/RolandM99/Setup-linter/settings/pages).
- Select the branch name
- Select root
- Click the Save button

Your project will be deployed by Github Pages

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED 🤝 

Hi @RolandM99,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**